modules/cnf-troubleshooting-missing-rte-config-maps.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-troubleshooting-missing-rte-config-maps_{context}"]
+= Correcting a missing resource topology exporter config map
+
+If you install the NUMA Resources Operator in a cluster where the cluster settings are misconfigured, in some circumstances, the operator is shown as active but the logs show that the resource topology exporter pods are missing, for example:
```

Comment:
"...shown as active but the logs of the RTE daemonset pods show that the configuration for the exporter is missing, for example:..."